### PR TITLE
First good looking header

### PR DESCRIPTION
**Pull request by exalted**

### CODE REVIEW

File: app/views/application/_header.html.erb [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
-</div>
+<header class="site-header clearfix" data-ux-header>
+  <div class="placeholder__logo"></div>
+  <div class="main-action-container clearfix">
+    <%= content_for :main_action %>
+  </div>
+  <div class="user-profile clearfix">
+    <div class="avatars-container clearfix">
+      <%= render partial: "users/avatar",
+                 object: current_user, as: :user,
```
Comment (as-cii) [inline app/views/application/_header.html.erb]: What about using a local for this as well?

```erb
<%= render partial: "users/avatar", locals: { 
  size: :small,
  user: current_user 
} %>
```

**Comment by as-cii** [comment]
One small comment, then :shipit:

File: app/views/application/_header.html.erb (hunk 1 above)
Comment (exalted) [inline app/views/application/_header.html.erb]: `users/_avatar.html.erb` is used as...

```
      <%= render partial: "users/avatar",
                 collection: board.collaborators, as: :user,
                 locals: { size: "tiny" } %>
```

... in https://github.com/catchup/catchup-web/pull/60/files#diff-641397f5b5168b3fed969d3a880d9358R13.

Do you think could it work anyway?

Comment (as-cii) [inline app/views/application/_header.html.erb]: Yup! :+1:

**Comment by exalted** [comment]
@as-cii please checkout 03bd7f7 whenever you can.

**Pull request merged**